crates/core/src/test_utils/deploy_helpers.rs

```diff
@@ -423,14 +443,31 @@ pub fn compute_kakarot_contracts_class_hash() -> Vec<(String, FieldElement)> {
         .collect()
 }
 
+fn compiled_kakarot_path() -> PathBuf {
```

Comment:
so this is kakarot build dir right? when the path is a directory, it's more explicit to use directory

Reply:
in the beginning I thought it was the full path like `kakarot/build/kakarot.json`